packages/framework/src/client.test.ts

```diff
@@ -272,7 +272,7 @@ describe('Novu Client', () => {
           }),
           {
             providers: {
-              slack: async ({ controls }) => {
+              slack: async () => {
```

Comment:
After adding type checking to Vitest, this test file complained about `controls` being implicit `any`. We weren't using it, so it's removed.